src/device-registry/config/constants.js

```diff
@@ -94,11 +90,13 @@ const defaultConfig = {
     weather_stations: "nearest-weather-stations",
   },
   KEY_ENCRYPTION_KEY: process.env.KEY_ENCRYPTION_KEY,
-  GET_ROAD_METADATA: ({ path, latitude, longitude } = {}) => {
-    const today = monthsInfront(0);
-    const oneMonthAgo = monthsInfront(-1);
-    const endDate = generateDateFormatWithoutHrs(today);
-    const startDate = generateDateFormatWithoutHrs(oneMonthAgo);
+  GET_ROAD_METADATA: ({
+    path,
```

Comment:
just modified the function to receive the variables of the values rather than having to require another module within the function